app/utils/ViewUtils.scala

```diff
@@ -16,19 +16,47 @@
 
 package utils
 
+import play.api.i18n.Messages
 import play.api.mvc.Call
 import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{HtmlContent, Text}
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist._
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.util.Locale
 
 object ViewUtils {
 
+  private val gbpCurrencySymbol = "£"
+
+  def bigDecimalCurrency(bigDecimal: BigDecimal): String = {
+    gbpCurrencySymbol + f"$bigDecimal%1.2f".replace(".00", "")
+  }
```

Comment:
We already have bigDecimalCurrency in the view helpers folder so this can be removed